aqua/MenuMeters/Portfile

```diff
@@ -17,18 +18,13 @@ long_description    The MenuMeters monitors are true SystemUIServer plugins
                     (also known as Menu Extras). This means they can be reordered   \
                     using command-drag and remember their positions in the menubar  \
                     across logins and restarts.
+
 homepage            http://www.ragingmenace.com/software/menumeters
-master_sites        http://www.ragingmenace.com/software/download
 distname            ${name}
 dist_subdir         ${name}/${version}
```

Comment:
Why would these two variables (`distname` and `dist_subdir`) still need to be set? I guess they can go, but didn't test yet.